js/OmniNav.js

```diff
@@ -4,7 +4,89 @@
 var OmniNav = (function() {
```

Comment:
This file probably contains the most important new stuff to pay attention to and comment on. Most of the other stuff in this pull request is removing old styles, replacing with v2 styles, updating HTML to reflect v2, etc.